src/core/term.ml

```diff
@@ -402,9 +404,10 @@ let cmp : term cmp =
     | Meta _ -> 7
     | Patt _ -> 8
     | TEnv _ -> 9
-    | Wild -> 10
+    | Wild   -> 10
     | TRef _ -> 11
     | LLet _ -> 12
+    | Plac _ -> 13
```

Comment:
We should simply remove this comment now I think.

Reply:
I'm not sure to understand what you mean  here.

Reply:
You can simply remove everything between lines 401 and 421.